plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -1814,4 +1814,30 @@ public function testStreamsRelationshipsManage()
             'You are not authorized to manage an object relationship to streams, please update stream relationship to objects instead'
         ));
     }
+
+    /**
+     * Test save emojis in text fields.
+     *
+     * @return void
+     * @coversNothing
+     */
+    public function testEmoji()
```

Comment:
Maybe this method should belong to `ObjectsTableTest`? Surely not controller related...